HPXMLtoOpenStudio/resources/hotwater_appliances.rb

```diff
@@ -1045,7 +1046,7 @@ def self.get_hwdist_recirc_pump_energy(hot_water_distribution, fixtures_usage_mu
     # Shared recirculation system pump energy
     # Assume the fixtures_usage_multiplier only applies for Sensor/Manual control type.
     if hot_water_distribution.has_shared_recirculation
-      n_dweq = hot_water_distribution.shared_recirculation_number_of_units_served
+      n_bdeq = [hot_water_distribution.shared_recirculation_number_of_bedrooms_served, nunits].max
```

Comment:
E.g., a 6-unit building having 3 units with 1 bedroom and 3 units with 0 bedrooms - We want the denominator to be 6 here.